scripts/globals/effects/defense_boost.lua

```diff
@@ -7,13 +7,22 @@ local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
     target:addMod(xi.mod.DEFP, effect:getPower())
+
+    if target:isInDynamis() then
```

Comment:
Cocoon from BLU grants this defense boost buff, I know it's not a problem now but be aware